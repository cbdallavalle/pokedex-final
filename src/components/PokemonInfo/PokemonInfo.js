import React from 'react';
import './PokemonInfo.css';

export const PokemonInfo = ({ icon, name, type, weight }) => {
  return (
    <div className="PokemonInfo">
      <p>{name}</p>
      <p>{type}</p>
      <p>{weight}</p>
      <img src={icon} alt={`${name} icon`} />
    </div>
  )
}