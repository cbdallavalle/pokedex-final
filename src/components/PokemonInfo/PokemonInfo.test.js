/* eslint-disable */
import React from 'react';
import { PokemonInfo } from './PokemonInfo';
import { shallow } from 'enzyme';

describe("PokemonInfo", () => {
  const props = {
    icon: '',
    name: '',
    type: '',
    weight: ''
  }

  it('should match the snapshot', () => {
    const wrapper = shallow(<PokemonInfo {...props} />)
    expect(wrapper).toMatchSnapshot();
  })
})