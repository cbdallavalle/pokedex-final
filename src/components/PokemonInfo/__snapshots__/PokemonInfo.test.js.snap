// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PokemonInfo should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    icon=""
    name=""
    type=""
    weight=""
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <p>
          
        </p>,
        <p>
          
        </p>,
        <p>
          
        </p>,
        <img
          alt=" icon"
          src=""
        />,
      ],
      "className": "pokemon-info",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "",
        },
        "ref": null,
        "rendered": "",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "",
        },
        "ref": null,
        "rendered": "",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "",
        },
        "ref": null,
        "rendered": "",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": " icon",
          "src": "",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <p>
            
          </p>,
          <p>
            
          </p>,
          <p>
            
          </p>,
          <img
            alt=" icon"
            src=""
          />,
        ],
        "className": "pokemon-info",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "",
          },
          "ref": null,
          "rendered": "",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "",
          },
          "ref": null,
          "rendered": "",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "",
          },
          "ref": null,
          "rendered": "",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": " icon",
            "src": "",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
